.github/CODEOWNERS

```diff
@@ -7,3 +7,5 @@
 .githooks/** @asynchroza
 
 /packages/web/src/components/Navigation/** @asynchroza @Aleks1908 @IvanObreshkov @DemonHashira @annakukova
+
+/packages/web/nginx.conf @IvanObreshkov
```

Comment:
add me as a codeowner as well

Reply:
furthermore, you may add yourself as a codeowner to the files you have edited in the api.